File: packages/cli/test/fixtures/ranking-corpora/java/src/main/java/com/example/ranking/handlers/CreateHandler.java
package com.example.ranking.handlers;

import com.example.ranking.util.Logger;

/** Validates and records a new user by name. */
public class CreateHandler {
    public void createUser(String name) {
        if (name == null || name.isEmpty()) {
            Logger.logError("createUser", new IllegalArgumentException("name must not be empty"));
            return;
        }
        Logger.logInfo("created user " + name);
    }
}

File: packages/cli/test/fixtures/ranking-corpora/kotlin/src/main/kotlin/com/example/ranking/handlers/CreateHandler.kt
package com.example.ranking.handlers

import com.example.ranking.util.Logger

/** Validates and records a new user by name. */
class CreateHandler {
    fun createUser(name: String) {
        if (name.isEmpty()) {
            Logger.logError("createUser", IllegalArgumentException("name must not be empty"))
            return
        }
        Logger.logInfo("created user $name")
    }
}
